resources/js/app.js

```diff
@@ -29,10 +35,22 @@ import 'bootstrap-vue/dist/bootstrap-vue.css'
 Vue.component('concept', require('./components/Concept.vue').default);
 Vue.component('concept-list', require('./components/ConceptList.vue').default);
 Vue.component('concept-form', require('./components/ConceptForm.vue').default);
-Vue.component('concept-category', require('./components/ConceptCategory.vue').default);
-Vue.component('concept-create', require('./components/ConceptCreate.vue').default);
-Vue.component('concept-source', require('./components/ConceptSource.vue').default);
-Vue.component('concept-search', require('./components/ConceptSearch.vue').default);
+Vue.component(
+  'concept-category',
+  require('./components/ConceptCategory.vue').default,
+);
+Vue.component(
+  'concept-create-form',
+  require('./components/Concept/CreateForm.vue').default,
+);
+Vue.component(
+  'concept-source',
+  require('./components/ConceptSource.vue').default,
+);
```

Comment:
Just a question for me: In my branch, I have created a folder called Concept as well to follow the pattern of `Concept/CreateForm.vue`. How do you decide what should be nested inside and what shouldn't? I've pretty much starting putting anything prefixed with `Concept` in there, but here you have `ConceptSource` outside the directory. Just curious what the expected structure should be so I can be consistent.

Reply:
So some of these are pre-existing work. I generally nest based on the domain language so Concept specific components would land in the `js/components/Concept` folder while generic components like buttons, modals, and other reusable bits of UI land at the top level of `js/components`. In a Vue 3 / Intertia powered Laravel frontend I would actually end up shifting those components out to something like `js/Pages/Concept/Partials/` and then the top level component for the interfaces would land at `js/Pages/Concept/{Show,Create,Edit}.vue` for example but I avoided wholesale re-architecting the application so we could keep moving.